### PR TITLE
Update README and documentation

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,18 +75,18 @@ pkg> add HarwellRutherfordBoeing
 
 julia> using HarwellRutherfordBoeing
 
-julia> matrix = ssmc[ssmc.name .== "bcsstk01", :]
+julia> matrices = ssmc[ssmc.name .== "bcsstk01", :]
```
Comment (dpo) [inline README.md:78]: but there is only one ;-)

File: docs/src/index.md [hunk 2]
```diff
@@ -5,20 +5,25 @@ A straightforward interface to the [SuiteSparse Matrix Collection](https://spars
...
-julia> for matrix in tiny
-         println(matrix_path(matrix, format="MM"))  # matrices are stored here
-       end
+julia> # select symmetric positive definite matrices with ≤ 100 rows and columns
+julia> tiny = ssmc[(ssmc.numerical_symmetry .== 1) .& (ssmc.positive_definite.== true) .& (ssmc.real .== true) .& (ssmc.nrows .≤ 100), :]
+
+julia> # fetch the matrices selects in MatrixMarket format
+julia> fetch_ssmc(tiny, format="MM")
+
+julia> matrix_paths(matrix, format="MM"))  # matrices are downloaded here
```
Comment (dpo) [inline docs/src/index.md:28]: Shouldn't it be `tiny` here too?

Comment (amontoison) [reply docs/src/index.md]: I forgot this one, thanks.

**Comment by dpo** [comment]
Thank you!

**Pull request merged**